### ISSUE TITLE
[Bug]: Warwick DC calendar not loading

### ISSUE THREAD

**Issue by jayhix123** [mampfes/hacs_waste_collection_schedule#3081]
### I Have A Problem With:

A specific source

### What's Your Problem

Calendar stopped loading in home assistant. Went to set it up again from fresh and it's reporting the following issue:

The source returned an invalid response: "time data 'Sat 04/01/2025' does not match format '%d/%m/%Y'". Please check the provided arguments and try again.
![Screenshot_20241123_210146_Home Assistant](https://github.com/user-attachments/assets/23db2099-ce6d-40a1-a517-9627ca092da2)

### Source (if relevant)

Warwick District Council

### Logs

```Shell
no relevant logs
```

### Relevant Configuration

```YAML
waste_collection_schedule:
    sources:
    - name: warwickdc_gov_uk
      args:
        uprn: *************
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [ ] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [X] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by jeverley** [comment]
Same issue here:

```
This error originated from a custom integration.

Logger: custom_components.waste_collection_schedule.waste_collection_schedule.source_shell
Source: custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py:160
integration: Waste Collection Schedule (documentation)
First occurred: 09:14:31 (1 occurrences)
Last logged: 09:14:31

fetch failed for source Warwick District Council: Traceback (most recent call last): File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 158, in fetch entries: Iterable[Collection] = self._source.fetch() ^^^^^^^^^^^^^^^^^^^^ File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/warwickdc_gov_uk.py", line 42, in fetch dates = [datetime.strptime(d.text, "%d/%m/%Y").date() for d in items[1:]] ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/usr/local/lib/python3.12/_strptime.py", line 554, in _strptime_datetime tt, fraction, gmtoff_fraction = _strptime(data_string, format) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/usr/local/lib/python3.12/_strptime.py", line 333, in _strptime raise ValueError("time data %r does not match format %r" % ValueError: time data 'Tue 07/01/2025' does not match format '%d/%m/%Y'
```

It would appear that for exceptional date changes (i.e. over christmas), the day name is returned in addition to the date:

![image](https://github.com/user-attachments/assets/7239569c-7c34-484e-a394-5a65af388b67)

**Comment by 5ila5** [comment]
Fixed by #3088

This will be part of the next release or is available now as master (git) version 

install master/git: Developer tools -> actions -> select `update: Install update` -> click `Coose entity` and select WCS -> select `version` and type `master` in the input field -> run the action -> restart HA


<details><summary>Older versions of HACS:</summary>
<p>
install master with HACS version < 2.0: (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)
</p>
</details>
